src/components/src/map/ai-assistant-control.tsx

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+// Copyright contributors to the kepler.gl project
+
+import React, {useCallback, ComponentType} from 'react';
+import {MapControls} from '@kepler.gl/types';
+import {AiStar} from '../common/icons';
+import {MapControlButton} from '../common/styled-components';
+import MapControlTooltipFactory from './map-control-tooltip';
+
+interface AiAssistantControlIcons {
```

Comment:
Nit: Any reason to use `interface` instead of `type` here, you seem to be mixing them.

Reply:
Add TSDoc `/** */` on the line before each new type and field? Costs little and integrates with vscode.